src/pipeline_edsl/semantics.ml

```diff
@@ -130,6 +130,10 @@ module type Bioinformatics_base = sig
     [ `Fastq ] repr ->
     [ `Optitype_result ] repr
 
+  val hlarp:
+    [`Optitype_result | `Seq2hla_result] repr ->
```

Comment:
So `_ repr` is not covariant this function AFAIK cannot be called.

Discuss at #318 :)